src/components/organisms/AuthenticationModal/LoginForm/LoginForm.tsx

```diff
@@ -44,9 +44,14 @@ const LoginForm: React.FunctionComponent<PropsType> = ({
 
   const onSubmit = async (data: LoginFormData) => {
     try {
-      if (CODE_CHECK_ENABLED) await axios.get(codeCheckUrl(data.code));
+      if (venue.requiresTicketCode) await axios.get(codeCheckUrl(data.code));
+      if (venue.requiresEmailVerification)
+        await axios.get(codeCheckUrl(data.email));
```

Comment:
Do we need any of the return values from this? Or is it fine to just catch errors?

Reply:
Yep we just catch errors so all good